### PR TITLE
🌐 [i18n-KO] Translated `reference/login.md` to Korean 

### PR DESCRIPTION
**Pull request by SeungAhSon**
# What does this PR do?

Translated the `reference/login.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
Team PseudoLab, may you please review this PR? 
@jungnerd, @heuristicwave, @harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

File: docs/source/ko/package_reference/login.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 로그인 및 로그아웃
```
Comment (jungnerd) [inline docs/source/ko/package_reference/login.md:5]:
```suggestion
# 로그인 및 로그아웃[[login-and-logout]]
```
Anchor를 추가헤주세요!

File: docs/source/ko/package_reference/login.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`huggingface_hub` 라이브러리를 사용하면 사용자가 허브에 기계를 프로그래밍 방식으로 로그인하고 로그아웃할 수 있습니다.
+
+인증에 대한 자세한 내용은 [이 섹션](../quick-start#authentication)을 확인하세요.
+
+## 로그인
```
Comment (SeungAhSon) [inline docs/source/ko/package_reference/login.md:11]:
```suggestion
## 로그인[[login]]
```

File: docs/source/ko/package_reference/login.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+[[autodoc]] login
+
+## 인터프리터_로그인
```
Comment (SeungAhSon) [inline docs/source/ko/package_reference/login.md:15]:
```suggestion
## 인터프리터_로그인[[interpreter_login]]
```

File: docs/source/ko/package_reference/login.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+[[autodoc]] interpreter_login
+
+## 노트북_로그인
```
Comment (SeungAhSon) [inline docs/source/ko/package_reference/login.md:19]:
```suggestion
## 노트북_로그인[[notebook_login]]
```

File: docs/source/ko/package_reference/login.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+[[autodoc]] notebook_login
+
+## 로그아웃
```
Comment (SeungAhSon) [inline docs/source/ko/package_reference/login.md:23]:
```suggestion
## 로그아웃[[logout]]
```

**Review by SeungAhSon — COMMENTED** [review]
Anchor 추가했습니다

File: docs/source/ko/package_reference/login.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 로그인 및 로그아웃
+
+`huggingface_hub` 라이브러리를 사용하면 사용자가 허브에 기계를 프로그래밍 방식으로 로그인하고 로그아웃할 수 있습니다.
```
Comment (seoulsky-field) [inline docs/source/ko/package_reference/login.md:7]:
```suggestion
`huggingface_hub` 라이브러리를 사용하면 사용자가 Hub에 기계를 프로그래밍 방식으로 로그인하고 로그아웃할 수 있습니다.
```

허브 -> Hub 국문 번역을 하지 않는 것으로 수정하였습니다.

**Review by seoulsky-field — COMMENTED** [review]
허브 -> Hub 국문 번역과 관련한 사항을 수정하였습니다!

File: docs/source/ko/package_reference/login.md (hunk 6 above)
Comment (junejae) [inline docs/source/ko/package_reference/login.md:7]:
```suggestion
`huggingface_hub` 라이브러리를 사용하면 사용자가 Hub에 기계를 프로그래밍적으로 로그인/로그아웃할 수 있습니다.
```
조금 풀어쓰는 표현인것 같아 축약해 보았습니다.

File: docs/source/ko/package_reference/login.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 로그인 및 로그아웃[[login-and-logout]]
+
+`huggingface_hub` 라이브러리를 사용하면 사용자가 Hub에 기계를 프로그래밍적으로 로그인/로그아웃할 수 있습니다.
```
Comment (jungnerd) [inline docs/source/ko/package_reference/login.md:7]:
```suggestion
`huggingface_hub` 라이브러리를 사용하면 사용자의 기기를 Hub에 프로그래밍적으로 로그인/로그아웃할 수 있습니다.
```
기계라는 말이 어색한 것 같아서 수정해보았어요!

**Comment by jungnerd** [comment]
@Wauplin LGTM 👍 I think this PR is good to go!

**Review by Wauplin — APPROVED** [review]
Awesome!

**Pull request merged**